kyo-core/shared/src/main/scala/kyo/streams.scala

```diff
@@ -182,88 +175,60 @@ object Stream:
         ): Stream[(T, T2), V, S & S2] =
             s.map(t => s2.map((t, _)))
 
-        def accumulate[S2, V2: Flat](init: V2)(f: (V2, V) => V2 < S2)(
```

Comment:
I've removed this method because I wasn't sure about its semantics. We can add it back later.